### DISCUSSION TITLE
ProxyState is not iterable error in panda website project that I set up locally

### DISCUSSION THREAD

**Discussion by muhsindeniz** [chakra-ui/panda#975] [Q&A] [answered]
Hello, as I want to contribute to the panda library designly, I forked the project and downloaded it in the website folder.

I did 'yarn install'.

Then I started the project and it gives error "TypeError: proxyState is not iterable".

What is the solution?

<img width="1060" alt="Screen Shot 2023-07-07 at 15 57 47" src="https://github.com/chakra-ui/panda/assets/49768007/2ab58a59-aa14-4353-a241-dc7ad88a9750">

**Comment by anubra266** [comment, marked as answer]
@muhsindeniz this error is not from panda your screenshot shows it's from `@zag-js/store` package
